.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "src/pm/hydra/tools/topo/hwloc/hwloc"]
 	path = src/pm/hydra/tools/topo/hwloc/hwloc
 	url = https://github.com/open-mpi/hwloc
+[submodule "confdb"]
+	path = confdb
+	url = git@github.com:pmodels/confdb.git
```

Comment:
I'm not sure I agree with this one. For all the others, there could conceivably be another version of UCX/hwloc/mpl that you might want to use or they could be used in other projects. Do we really expect to have another version of confdb or it be usable elsewhere? There's a lot of MPICH-isms in there.